akka-http2-support/src/test/scala/akka/http/impl/engine/http2/RequestParsingSpec.scala

```diff
@@ -0,0 +1,540 @@
+package akka.http.impl.engine.http2
```

Comment:
I know I'm boring with those comments about the header: each file needs copyright header, see other files. :)

Reply:
OK good catch.

PS: Play has a copyright header check as part of the build. I'll can add a header check to akka-http in another PR.

Reply:
Re:PS: That would be cool; We would like to use Heiko's header adder at some point, but just failing is great too 👍  The header plugin we could not use for some reason back in the past, but supposedly the new version should work...

Thanks in advance :)